tests/Faithlife.Analyzers.Tests/VerbatimStringTests.cs

```diff
@@ -0,0 +1,79 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Framework;
+
+namespace Faithlife.Analyzers.Tests
+{
+	[TestFixture]
+	public sealed class VerbatimStringTests : DiagnosticVerifier
+	{
+		[Test]
+		public void ValidStrings()
+		{
+			const string validProgram = @"
+namespace TestApplication
+{
+	public class TestClass
+	{
+		public TestClass()
+		{
+			@""Hello	World"".Trim();
```

Comment:
Why is this considered a "valid" usage? Isn't the string the same without the `@`?

Reply:
For some reason, I thought you had to use `\t` in regular strings... But apparently you don't, you can just type tabs. TIL. Since tabs are unchanged between verbatim and regular strings, I'll stop checking for them.